openfl-workspace/torch_cnn_mnist/plan/cols.yaml

```diff
@@ -1,5 +1,6 @@
-# Copyright (C) 2020-2021 Intel Corporation
+# Copyright (C) 2024 Intel Corporation
```

Comment:
2020-2024?

Reply:
resolved in d8dda083b7c32d0680ef2d87564c3fa64b888f19